dusty/compiler/nginx/__init__.py

```diff
@@ -1,7 +1,7 @@
-from ...constants import NGINX_MAX_FILE_SIZE
+from ...constants import NGINX_MAX_FILE_SIZE, VM_IP_FROM_DOCKER
 
 def _nginx_proxy_string(port_spec):
-    return "proxy_pass http://{}:{};".format(port_spec['boot2docker_ip'], port_spec['proxied_port'])
+    return "proxy_pass http://{}:{};".format(VM_IP_FROM_DOCKER, port_spec['proxied_port'])
```

Comment:
It's the IP of the VM (like, non-containerized VM TCL host) if you're a container. I should really draw up a diagram of how this works now and throw it in the docs.

The Nginx and target containers both forward ports to the VM, then we do this to make it all tie together:

Mac -> VM -> Nginx container -> VM again -> Target container